### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
I removed outdated information in the installation instructions, both the `setup.py` approach for installation, as well as information on Windows install, since we do not currently support that.

I am also doing this partly to trigger a PR build for readthedocs, and testing that feature. 

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- Documentation update

### CODE REVIEW

**Comment by nwu63** [comment]
No reason at all. The only place where we need to talk about pyOpt is on the page describing the differences, and also in the README.

Would you mind going through and updating that?  You should be able to directly push to the branch. I can do this also if you don't have time.

**Comment by sseraj** [comment]
Sure, I can do that.

**Comment by sseraj** [comment]
Installing with pip will uninstall previous versions. Should we still keep the line about uninstalling before installing a new version? There are no uninstall instructions.

**Comment by nwu63** [comment]
That's a good point, it's a bit more nuanced than that. Once you update the other stuff I'll edit the file again.

File: doc/install.rst [hunk 1]
```diff
@@ -33,11 +33,22 @@ For those not using ``conda``, a user install is needed::
...
-    
-* You may want to uninstall any previous version of pyOptSparse before installing a new
-  version, as there may be conflicts.
+
 * Some optimizers are licensed and their sources are not included with this distribution. 
   To use them, please request their sources from the authors as indicated in the optimizer 
   LICENSE files, and place them in their respective source folders before installing the package.
   Refer to specific optimizer pages for additional information.
+
+Update or Unintall
```
[the remark this answers is not in the document]
Comment (nwu63) [reply doc/install.rst:44]: Ah good stuff

**Pull request merged**